src/test/java/hudson/plugins/junitattachments/AttachmentPublisherPipelineTest.java

```diff
@@ -100,7 +101,7 @@ public void annotationDoesNotFailForPipeline() throws Exception {
     private TestResultAction getTestResultActionForPipeline(String workspaceZip, String pipelineFile, Result expectedStatus) throws Exception {
         WorkflowJob project = jenkinsRule.jenkins.createProject(WorkflowJob.class, "test-job");
         FilePath workspace = jenkinsRule.jenkins.getWorkspaceFor(project);
-        FilePath wsZip = workspace.child("workspace.zip");
+        FilePath wsZip = Objects.requireNonNull(workspace).child("workspace.zip");
```

Comment:
Why did you do this? It's not clear

Reply:
to check workspace for null

Reply:
Unnecessary since if it _were_ null, the test would simply fail with a `NullPointerException` anyway.

(In `src/main/**/*.java` when a `@CheckForNull` method is being called, we would of course want to think about what a null result could mean and fail with a user-appropriate message, or recover gracefully where that makes sense.)